assets/js/modal.js

```diff
@@ -0,0 +1,151 @@
+const modal = async (id) => {
```

Comment:
```suggestion
const productModal = async (id) => {
```
Chỗ này mình đặt tên cho tường minh 1 chút luôn Hiển, để tránh bị nhầm với modal khác